inloop/statistics/truncators.py

```diff
@@ -0,0 +1,19 @@
+import django.db.models.functions as functions
+
+
+class GranularityUnavailable(ValueError):
+    pass
+
+
+def get_truncator(field_name: str, granularity: str):
```

Comment:
Why not use a `dict`?

Reply:
I did not use a dictionary in this case, because the truncator depends on the field name to work properly. But instead of the if clauses, I could of course use a dict inside the function itself. Will do 👍🏻